hub/constraints/miscellaneous-rows/oob.lisp

```diff
@@ -156,6 +165,7 @@
                                         (eq!    (shift [ misc/OOB_DATA 6 ]    kappa)   call_stack_depth )
                                         ;; (eq!    (shift [ misc/OOB_DATA 7 ]    kappa) )    ;; value_is_nonzero
                                         ;; (eq!    (shift [ misc/OOB_DATA 8 ]    kappa) )    ;; aborting condition
+                                        (eq!    (shift [ misc/OOB_DATA 9 ]    kappa)   creator_nonce    )
```

Comment:
Here is the modification to the OOB_INST_CREATE @letypequividelespoubelles , is this what you are looking for ?